src/middleware/get-payload.ts

```diff
@@ -4,13 +4,18 @@ import AggregateError from "aggregate-error";
 import { IncomingMessage } from "http";
 import { EventTypesPayload } from "../generated/get-webhook-payload-type-from-event";
 
+declare module "http" {
+  interface IncomingMessage {
+    body?: EventTypesPayload;
+  }
+}
```

Comment:
could this become a problem if people use the `IncomingMessage` interface in an app that uses `@octokit/webhooks`? I'd be cautious to pollute global name space? Doesn't that have similar problems to pollution Object prototypes in JavaScript?

Reply:
> Doesn't that have similar problems to pollution Object prototypes in JavaScript?

Yes and no - ultimately this is about making the types more accurate so it should be fine, but it's usually not something you want to be doing a whole lot for similar reasons to polluting object prototypes.

But because of how TypeScript loads types, it'll only "see" this if you import the file (i.e importing `getPayload`), at which point it becomes structurally accurate since it's possible for `IncomingMessage` to have an optional `body` property on it of a particular type, and so this enables setting that property on such a type.

(You do raise a good point: this is something very much to keep in mind since this is in the "library" space instead of the "app" space, and is why I closed #404 - I'm happy to revert it if it does end up causing trouble for someone)